packages/core/upload/server/controllers/validation/admin/folder.js

```diff
@@ -1,25 +1,27 @@
 'use strict';
 
+const { isUndefined, get } = require('lodash/fp');
 const { yup, validateYupSchema } = require('@strapi/utils');
-const { isNil } = require('lodash/fp');
 const { getService } = require('../../../utils');
+const { folderExists } = require('./utils');
 
+const folderModel = 'plugin::upload.folder';
```

Comment:
I wonder if it is about time to put the var into constants since it is now referenced in many places?

Reply:
yes 👌